src/articles/guides/tools.md

```diff
@@ -32,6 +34,14 @@ Familiarizing yourself with powerful tools will streamline your workflow and hel
 
 -   [fugitive](https://github.com/tpope/vim-fugitive) - A plugin for Vim that wraps Git to allow easier command usage
 
+## Shell Scripting
```

Comment:
```suggestion
## Bash
```

Reply:
if you write a script for bash, will it run for other shells?